.github/workflows/ruby.yml

```diff
@@ -0,0 +1,31 @@
+# This workflow will download a prebuilt Ruby version, install dependencies and run tests with Rake
+# For more information see: https://github.com/marketplace/actions/setup-ruby-jruby-and-truffleruby
+
+name: Ruby
```

Comment:
Small comment, this name will show up in the badge as [Ruby | passing]. https://github.com/salemove/salestation/blob/1e62a60ab0b33c9d0c72df2f8a3a39e989a5cb31/README.md
So might want to rename to something like `CI` or `tests` or `build` (how it was).

Reply:
```suggestion
name: CI
```